owncloudApp/src/main/res/layout/list_fragment.xml

```diff
@@ -18,80 +18,117 @@
 -->
 <androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:fab="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/coordinator_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:filterTouchesWhenObscured="true">
 
-    <FrameLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:layout_gravity="top">
 
-        <androidx.swiperefreshlayout.widget.SwipeRefreshLayout
-            android:id="@+id/swipe_containing_list"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:footerDividersEnabled="false"
-            android:visibility="visible">
+        <com.owncloud.android.presentation.ui.files.SortOptionsView
+            android:id="@+id/options_layout"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            fab:layout_constraintEnd_toEndOf="parent"
+            fab:layout_constraintStart_toStartOf="parent"
+            fab:layout_constraintTop_toTopOf="parent" />
 
-            <com.owncloud.android.ui.ExtendedListView
-                android:id="@+id/list_root"
-                android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                android:divider="@color/list_divider_background"
-                android:dividerHeight="1px"
-                android:visibility="visible" />
-        </androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
+        <ProgressBar
+            android:id="@+id/syncProgressBar"
+            style="@style/Widget.ownCloud.TopProgressBar"
+            android:layout_width="match_parent"
+            android:layout_height="4dp"
```

Comment:
Do not hardcode dimens

Reply:
Sure, moving old stuff.. i forgot to update it 🤔